lib/sycamore/sycamore/query/operators/count.py

```diff
@@ -10,12 +10,12 @@ def __init__(self, node_id: str, data: Optional[Dict[Any, Any]] = None) -> None:
     @staticmethod
     def description() -> str:
         return """
-        **Count**: Determines the length of a particular data table. Parameters are *description*,
+        **Count**: Determines the length of a particular database (number of records). Parameters are *description*,
```

Comment:
I'm curious about the switch from 'data table' to 'database' here and below. Does this help?

Reply:
I actually don't know if it helps, changed it mostly for normalization purposes. However, I wonder if it will help as we get into more database-y operations like joins and aggregations.